package.json

```diff
@@ -12,37 +12,50 @@
     "react-helmet": "^5.2.0",
     "react-router": "^4.1.2",
     "react-router-dom": "^4.1.2",
-    "react-scripts": "1.0.11",
     "semantic-ui-css": "2.2.12",
     "semantic-ui-react": "^0.75.1"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
-    "coverage": "react-scripts test --coverage",
+    "test": "cross-env CI=true react-scripts test",
+    "test:coverage": "cross-env CI=true react-scripts test --coverage",
     "eject": "react-scripts eject",
-    "deploy": "npm run build&&gh-pages -d build",
-    "generateSNAPTests": "./node_modules/.bin/babel-node src/test/programs/federal/snap/generateTestCases",
-    "lint": "./node_modules/.bin/eslint src --fix"
+    "deploy": "run -s build gh-pages",
+    "gh-pages": "gh-pages -d build/",
+    "generateSNAPTests": "babel-node src/test/programs/federal/snap/generateTestCases",
+    "lint:eslint": "eslint src",
+    "lint:prettier": "prettier --list-different '**/*.{md,json}'",
+    "fix:eslint": "eslint src --fix",
+    "fix:prettier": "prettier --write '**/*.{md,json}'"
```

Comment:
Nice distinction. Maybe additional scripts `lint` and `lint-fix` could run both eslint and prettier (if we're going to keep prettier).

Reply:
Added~